mlrun/api/api/endpoints/tags.py

```diff
@@ -23,15 +23,16 @@
 import mlrun.api.schemas
 import mlrun.api.utils.auth.verifier
 import mlrun.api.utils.singletons.project_member
+from mlrun.utils.helpers import tag_name_regex_as_string
 
 router = fastapi.APIRouter()
 
 
 @router.post("/projects/{project}/tags/{tag}", response_model=mlrun.api.schemas.Tag)
 async def overwrite_object_tags_with_tag(
     project: str,
-    tag: str,
```

Comment:
Is the change in order was really needed? I understand that now that we set a default we would need to add default to the other params, but I think intuitively and with our current convention that we first define the path parameters and afterwards the query params and then the body.